addons/mail/wizard/mail_wizard_invite.py

```diff
@@ -61,23 +43,51 @@ def add_followers(self):
             document.message_subscribe(partner_ids=new_partners.ids)
 
             model_name = self.env['ir.model']._get(wizard.res_model).display_name
-            # send a notification if option checked and if a message exists (do not send void notifications)
-            if wizard.notify and wizard.message and not is_html_empty(wizard.message):
-                message_values = wizard._prepare_message_values(document, model_name, email_from)
-                message_values['partner_ids'] = new_partners.ids
-                document.message_notify(**message_values)
+            message_values = wizard._prepare_message_values(document, model_name, user_from)
+            message_values['partner_ids'] = wizard.partner_ids.ids
+            message_values['subtype_xmlid'] = "mail.mt_comment"
```

Comment:
Would add those in "prepare message values", that way it really prepare al values ^^